lib/src/service.dart

```diff
@@ -44,6 +44,13 @@ class ServiceDescription {
     }
   }
 
+  static Uri patchUrl(Uri uri, String path) {
```

Comment:
Could you maybe add a comment/docs what this method does? From the name only it is not clear whats happening here.

Reply:
Yes, comment added. Also add more test.

Please review again.